src/ssl/cert_validate_message.cc

```diff
@@ -16,22 +16,36 @@
 #include "ssl/support.h"
 #include "util.h"
 
+/// Retrieves the certificates chain used to verify the peer
+/// This is the full chain built by openSSL while verifying the server
+/// certificate or, if this is not available, the chain sent by server.
+/// \return the certificates chain or null
```

Comment:
s/or null/or nil/ to avoid complaints that "null" is too similar to the prohibited "NULL" or that it is too similar but not quite "nullptr".